### PR TITLE
Feature/add classes

### PR DESCRIPTION
**Pull request by pinkmoon25**
### OOP school library: add Person, Student and Teacher classe

- Implement classes and objects in Ruby.
- Implement encapsulation and inheritance with Ruby.
- Created parent class Person
- Created child classes Student and Teacher

### CODE REVIEW

**Review by DammyShittu — COMMENTED** [review]
Hi @PinkMoon25,

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

### You do not have a descriptive readme documentation

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1,2 +1,53 @@
...
+
+To get a working local copy up and running follow these simple steps
+
+- open terminal
+- clone this project with the command:
+
+```
+git clone https://github.com/PinkMoon25/school-library.git
+
+```
```
Comment (danmainah) [inline README.md:26]: - [ ] [optional] It would have been nice adding how to run the project locally.

**Review by danmainah — APPROVED** [review]
Hi @PinkMoon25 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights

✔️ No linter errors
✔️ Github flow was correctly used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by pinkmoon25** [comment]
Thanks, @danmainah for the review :+1:

**Pull request merged**